setup.py

```diff
@@ -34,9 +34,7 @@
     ],
     # Using older jq on amd64 due to https://github.com/reanahub/reana-demo-bsm-search/issues/21
```

Comment:
```suggestion
```
This line can be removed, as now we are using the same new version of jq for both amd64 and arm64